types/auth0/auth0-tests.ts

```diff
@@ -191,26 +191,60 @@ auth.tokens
     });
 
 // Password Grant
-auth.passwordGrant({username: 'username', password: 'password'}).then((response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, (err, response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, { forwardedFor: '12.34.56.78' }).then((response: auth0.TokenResponse) => { console.log(response); });
-auth.passwordGrant({username: 'username', password: 'password'}, { forwardedFor: '12.34.56.78' }, (err, response: auth0.TokenResponse) => { console.log(response); });
+auth.passwordGrant({ username: 'username', password: 'password' }).then((response: auth0.TokenResponse) => {
+    console.log(response);
+});
+auth.passwordGrant({ username: 'username', password: 'password' }, (err, response: auth0.TokenResponse) => {
+    console.log(response);
+});
+auth.passwordGrant({ username: 'username', password: 'password' }, { forwardedFor: '12.34.56.78' }).then(
+    (response: auth0.TokenResponse) => {
+        console.log(response);
+    },
+);
+auth.passwordGrant(
+    { username: 'username', password: 'password' },
+    { forwardedFor: '12.34.56.78' },
+    (err, response: auth0.TokenResponse) => {
+        console.log(response);
+    },
+);
 
 // SMS/Email OTP Login
-auth.requestEmailCode({email: 'hi@me.co', authParams: {}}).then((response: any) => { console.log(response); });
-auth.requestEmailCode({email: 'hi@me.co', authParams: {}}, (response: any) => { console.log(response); });
+auth.requestEmailCode({ email: 'hi@me.co', authParams: {} }).then((response: any) => {
+    console.log(response);
+});
+auth.requestEmailCode({ email: 'hi@me.co', authParams: {} }, (response: any) => {
+    console.log(response);
+});
```

Comment:
```suggestion
auth.requestSMSCode({ phone_number: '+1234567890' }, error => {
    error; // $ExpectType Error
});
auth.requestSMSCode({ phone_number: '+1234567890' }).then(error => {
    error; // $ExpectType any
});
```
and same in other places, please do not introduce just formatting changes. If those changes derive from PR focus, move them into new one